src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -1,186 +1,239 @@
 import React from 'react';
 import { expect } from 'chai';
 import moment from 'moment';
-import { Route } from 'react-router-dom';
-import { fireEvent } from '@testing-library/react';
 import environment from 'platform/utilities/environment';
-import {
-  mockFetch,
-  resetFetch,
-  setFetchJSONResponse,
-} from 'platform/testing/unit/helpers';
-
-import RequestedAppointmentDetailsPage from '../../../appointment-list/components/RequestedAppointmentDetailsPage';
-import { renderWithStoreAndRouter } from '../../mocks/setup';
-import {
-  getVAFacilityMock,
-  getVARequestMock,
-  getMessageMock,
-} from '../../mocks/v0';
-import { FETCH_STATUS } from '../../../utils/constants';
+import { setFetchJSONFailure } from 'platform/testing/unit/helpers';
```

Comment:
Are the changes in this unit test changes you made? They seem unrelated to the page you're adding.

Reply:
No. I copied the current version of master to my branch to resolve some merge conflicts.